tools/generate_package_config/pubspec.yaml

```
@@ -0,0 +1,6 @@
+name: generate_package_config
```

it is probably worth putting an sdk constraint in here so this doesn't get broken by the null safety release

```suggestion
name: generate_package_config
environment:
  sdk: ">=2.8.0 <3.0.0"
```

---

Done